### DISCUSSION TITLE
Are there any JS and CSS inclusive page source grabbers available online

### DISCUSSION THREAD

**Discussion by ilovetoucans** [3kh0/ext-remover#954] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Online ive been able to find dozens of unblocked page source grabbers and for just viewing stuff on websites that are blocked its useful, but when your looking to do something such as interact with the page with events only created with java script it messes up the whole thing accompanied by the ugliness of the page caused by not having CSS put in the page source.

**Comment by v4pl** [comment, marked as answer]
I know right I been trying to do it too this one called httrack is good but doesn't capture js.

**Reply by ilovetoucans** [reply]
Thanks for the software recommendation! Unrelated but it's definitely not often when I see downloadable software with a bsd version
